.rubocop.yml

```diff
@@ -11,6 +11,7 @@ AllCops:
     - 'db/**/*'
     - 'config/**/*'
     - 'script/**/*'
+    - 'vendor/**/*'
```

Comment:
without this it parses the sourcecode of the gems in your `vendor/bundle` :see_no_evil: